### PR TITLE
test: unit tests for podmanengine.go file in container package

### PR DESCRIPTION
**Pull request by tarun8718**
This PR consists of unit tests for podmanengine.go file in the container package.

Below are the tests and their test execution status:
image

### CODE REVIEW

File: environment/container/podmanengine.go [hunk 1]
```diff
@@ -38,15 +38,15 @@ func newPodmanEngine() *podmanEngine {
...
 	inspectcmd := exec.Command("podman", "inspect", image)
 	logrus.Debugf("Inspecting image %s", image)
 	output, err := inspectcmd.CombinedOutput()
 	if err != nil {
 		logrus.Debugf("Unable to inspect image %s : %s, %s", image, err, output)
-		return t, err
+		return nil, err
 	}
-	t = types.ImageInspect{}
+	var t []types.ImageInspect
```
Comment (tarun8718) [inline environment/container/podmanengine.go:49]: Inspect functionality in podman returns an array of json, which can only be unmarshalled into a slice of structs.

Fixes this: 
Screenshot 2023-03-26 at 11 37 29 PM

**Comment by HarikrishnanBalagopal** [comment]
> Inspect functionality in podman returns an array of json, which can only be unmarshalled into a slice of structs.
> 
> Fixes this: Screenshot 2023-03-26 at 11 37 29 PM

It has to match this interface https://github.com/konveyor/move2kube/blob/6156882d26f8b4c61b55d8488a0e20da6b54ba62/environment/container/container.go#L44

**Comment by tarun8718** [comment]
> https://quay.io/repository/konveyor/hello-world

@ashokponkumar some tests (like run cmd in a container) requires functionalities like touch/sleep infinity; Such tests can't be run on hello-world as it doesn't have the support. In such case which image do I use?

**Review by HarikrishnanBalagopal — APPROVED** [review]
LGTM

**Pull request merged**